amr-wind/turbulence/LES/Kosovic.cpp

```diff
@@ -0,0 +1,195 @@
+#include <cmath>
+
+#include "amr-wind/turbulence/LES/Kosovic.H"
+#include "amr-wind/turbulence/TurbModelDefs.H"
+#include "amr-wind/fvm/nonLinearSum.H"
+#include "amr-wind/fvm/strainrate.H"
+#include "amr-wind/fvm/divergence.H"
+#include "AMReX_REAL.H"
+#include "AMReX_MultiFab.H"
+#include "AMReX_ParmParse.H"
+
+namespace amr_wind {
+namespace turbulence {
+
+template <typename Transport>
+// cppcheck-suppress uninitMemberVar
+Kosovic<Transport>::Kosovic(CFDSim& sim)
+    : TurbModelBase<Transport>(sim)
+    , m_vel(sim.repo().get_field("velocity"))
+    , m_rho(sim.repo().get_field("density"))
+    , m_Nij(sim.repo().declare_field("Nij", 9, 1, 1))
+    , m_divNij(sim.repo().declare_field("divNij", 3, 1, 1))
+{
+    amrex::ParmParse pp("Kosovic");
+    pp.query("Cb", m_Cb);
+    m_Cs = std::sqrt(8 * (1 + m_Cb) / (27 * 3.14 * 3.14));
```

Comment:
is 3.14 a stand-in for pi? If so, let's use the real pi value instead, by using "M_PI".